src/command-parser/parsers.ts

```diff
@@ -39,8 +42,16 @@ export const parseBIP32Path = (
   throw Error(Errors.InvalidPathError)
 }
 
-export const parseFileTypeMagic = (fileTypeMagic: string, path: string) => {
+export const parseFileTypeMagic = (fileTypeMagic: string, pathType: PathTypes) => {
   if (fileTypeMagic.startsWith('Payment')) {
+    // TODO cardano-cli doesn't support 'Mint' prefix now, but maybe it will
```

Comment:
My take on what cardano-cli is doing/will do:
they distinguish pool cold keys (only as a check against shooting oneself in the foot), but bundle all the rest into "payment" keys, i.e. they won't ever distinguish between 1852, 1854, 1855 etc., because they want to keep the tool generic, and keys are just keys, the way how they are created/derive does not matter to cardano-cli. Let's put this in a nicer way into a comment here and not leave any TODO.

Reply:
Also, there is another TODO just below, let's deal with it now somehow.

Reply:
Fixed in https://github.com/vacuumlabs/cardano-hw-cli/pull/98/commits/ce8f1651cd61e4ef036063db1348fdce61d352d4

Reply:
Fixed in https://github.com/vacuumlabs/cardano-hw-cli/pull/98/commits/ce8f1651cd61e4ef036063db1348fdce61d352d4 and https://github.com/vacuumlabs/cardano-hw-cli/pull/98/commits/09414b944a586888bc9b10df322fafd66b238322